h/views/api/exceptions.py

```diff
@@ -36,7 +36,7 @@ def api_notfound(request):
     return {"status": "failure", "reason": message}
 
 
-@json_view(context=APIError, decorator=cors_policy)
+@json_view(context=httpexceptions.HTTPError, decorator=cors_policy)
```

Comment:
This means this'll now catch any HTTPErrors raised, rather than just APIErrors as before.

Reply:
I've done fairly careful manual testing of various exceptions raised by API views and it all looks fine to me. See commit messages for details.